apollo/src/graphql/model-version/modelVersionQueries.ts

```diff
@@ -17,4 +17,20 @@ builder.queryFields((t) => ({
             return mlModelVersions;
         },
     }),
+    getMLModelVersion: t.field({
+        type: MLModelVersion,
+        authScopes: {
+            loggedIn: true,
+        },
+        args: {
+            modelVersionId: t.arg.string({ required: true }),
+        },
+        async resolve(_root, args, _ctx) {
+            const mlModelVersion = (await ObjectionMLModelVersion.query().findById(
+                args.modelVersionId,
+            )) as typeof MLModelVersion.$inferType;
```

Comment:
How come we need the inferType bit here? (Just curious, my brain hasn't turned on yet)

Also sorry -- didn't realize I hadn't implemented this query already 😬

Reply:
1) No worries broimgo.
2) You don't. I just copied and pasted it from the `getMLModel` query which also uses the infer type. You don't need it for other but for sake of consistency I left it. I'll remove from both if you prefer.

Reply:
Gotcha gotcha, alles gut. I was probably doing something dumb previously 😅

Reply:
Actually, it was probably a Nexus thing! I've noticed that Pothos has much better type inference.